dadguide/database_context.py

```diff
@@ -45,6 +48,8 @@ def get_monsters_by_active(self, active_skill_id: int, *, server: Server):
         return self.get_monsters_where(lambda m: m.active_skill_id == active_skill_id, server=server)
 
     def get_all_monster_ids_query(self, server: Server):
+        if self.debug_mons is not None:
```

Comment:
put a comment here explaining that if there's debug monsters then we are in debug mode, which means we're only working with an explicit subset of the entire monster list so we don't bother with the query, instead using just what the user provided